src/sql/azdata.d.ts

```diff
@@ -3086,9 +3086,16 @@ declare module 'azdata' {
 		ariaColumnCount?: number;
 		ariaRole?: string;
 		focused?: boolean;
+		checked?: CheckBoxInfo[];
```

Comment:
IIUC, you want to do partial data update. I'd recommend not to do this as this is very specific to checkbox. I think we should introduce a method updateCells(cellInfos:<row, column, value>[]). for now you only need to implement the checkbox related logic.

Reply:
I thought there wasn't a way for an extension to call a method of a component. I can change this property to a more generic one for all TableCells, but is that what you meant?